### PR TITLE
Setup vagrant

### PR DESCRIPTION
**Pull request by MCGallaspy**
Should work OK. Forwards to port 8030 because that's one I wasn't using.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "ubuntu/trusty32"
```
Comment (aronasorman) [inline Vagrantfile]: Please add `box_url` (see distributed's `Vagrantfile`)

File: centralserver/settings.py [hunk 2]
```diff
@@ -49,15 +49,27 @@
 LOCALE_PATHS   = getattr(local_settings, "LOCALE_PATHS", (PROJECT_PATH + "/../locale",))
 LOCALE_PATHS   = tuple([os.path.realpath(lp) + "/" for lp in LOCALE_PATHS])
 
-DATABASES      = getattr(local_settings, "DATABASES", {
+#DATABASES      = getattr(local_settings, "DATABASES", {
```
Comment (aronasorman) [inline centralserver/settings.py]: Remove commented out code.

File: Vagrantfile (hunk 1 above)
Comment (MCGallaspy) [inline Vagrantfile]: What's the benefit? [The docs](http://docs.vagrantup.com/v2/vagrantfile/machine_settings.html) suggest it's not necessary.

Comment (aronasorman) [inline Vagrantfile]: Oh yeah, looks like it isn't anymore.

**Pull request merged**